block-storage-attacher/tests/e2e/conf/prod-us-east.conf

```diff
@@ -67,5 +67,10 @@ TEST_CODE_BUILD=true
 TEST_SUITE_RUN=basic
 
 # Testcase variables
-PLUGIN_NAME=ibm.io/ibmc-block
+PLUGIN_NAME=ibm.io/ibmc-blockvolume-attacher
```

Comment:
Need to check where is this variable used and then confirm if value is correct.

Reply:
Here below we use this value 
block-storage-attacher/tests/e2e/framework/framework.go:        pluginName := os.Getenv("PLUGIN_NAME")
block-storage-attacher/tests/e2e/common/common_util.go: pluginNameEnv := os.Getenv("PLUGIN_NAME")

To print the plugin name while publishing the result

Reply:
Use `ibm.io/ibmc-blockattacher` as PLUGIN_NAME